frontend/src/modules/report/components/templates/report-template-filters.vue

```diff
@@ -120,28 +116,91 @@ const platformLabel = computed(() => {
   return props.platform.value.map((v) => v.label).join(', ');
 });
 
+const platformOptions = (platform) => CrowdIntegrations.getConfig(platform);
```

Comment:
For the platforms query params, do you think we can follow the most common approach? So for multiple platforms instead of doing this:
```
selectedPlatforms=discord+slack
```
we could have
```
&selectedPlatforms=discord&selectedPlatforms=slack
```

Reply:
We definitely can use the above approaches the current approach just felt easier on the eyes to look at the selected platforms in a single key value param, what's your say @joanagmaia

Reply:
Have made the change @joanagmaia now the selected platforms are separated by commas